recipe/build.sh

```diff
@@ -8,7 +8,8 @@ else
     EXE_SQLITE3=${BUILD_PREFIX}/bin/sqlite3
 fi
 
-cmake ${CMAKE_ARGS} \
+echo "Existing cmake args: ${CMAKE_ARGS}"
+cmake ${CMAKE_ARGS} -O0 \
       -D CMAKE_BUILD_TYPE=Release \
```

Comment:
I don't think cmake accepts -O0. That's a gcc flag. Try instead:
```suggestion
cmake ${CMAKE_ARGS} \
      -D CMAKE_BUILD_TYPE=Debug \
```

Reply:
Yeah I woke up, read your comment, made a change thinking `CMAKE_FLAGS` was `CFLAGS`, and then realized the mistake later. I defined CFLAGS and CXXFLAGS and still got the failure. I'll remove those and change the build type now.

Reply:
Still failing. Now I'll try lowering compiler and dependency versions.